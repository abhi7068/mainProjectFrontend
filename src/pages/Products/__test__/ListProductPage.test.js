import React from 'react';

import ListProductPage from '../ListProductsPage';
import { render } from '@testing-library/react';

it('it renders', () => {
  render(<ListProductPage />);
});
it('should take a snapshot', () => {
  const { asFragment } = render(<ListProductPage />);
  expect(asFragment(render(<ListProductPage />))).toMatchSnapshot();
});
