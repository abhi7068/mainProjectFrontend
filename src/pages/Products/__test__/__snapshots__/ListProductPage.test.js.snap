// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should take a snapshot 1`] = `
<DocumentFragment>
  <div>
    <div
      class="MuiTableContainer-root"
    >
      <table
        class="MuiTable-root"
        style="margin-top: 7%; margin-left: 250px; width: 75.8%;"
      >
        <thead
          class="MuiTableHead-root"
        >
          <tr
            class="MuiTableRow-root MuiTableRow-head"
          >
            <th
              class="MuiTableCell-root MuiTableCell-head WithStyles(ForwardRef(TableCell))-head-34"
              scope="col"
            >
              Product Name
            </th>
            <th
              class="MuiTableCell-root MuiTableCell-head WithStyles(ForwardRef(TableCell))-head-34 MuiTableCell-alignCenter"
              scope="col"
            >
              Description
            </th>
            <th
              class="MuiTableCell-root MuiTableCell-head WithStyles(ForwardRef(TableCell))-head-34 MuiTableCell-alignCenter"
              scope="col"
            >
              Price
            </th>
            <th
              class="MuiTableCell-root MuiTableCell-head WithStyles(ForwardRef(TableCell))-head-34 MuiTableCell-alignCenter"
              scope="col"
            >
              Discount
            </th>
            <th
              class="MuiTableCell-root MuiTableCell-head WithStyles(ForwardRef(TableCell))-head-34 MuiTableCell-alignCenter"
              scope="col"
            >
              Category
            </th>
            <th
              class="MuiTableCell-root MuiTableCell-head WithStyles(ForwardRef(TableCell))-head-34 MuiTableCell-alignCenter"
              scope="col"
            >
              Edit
            </th>
            <th
              class="MuiTableCell-root MuiTableCell-head WithStyles(ForwardRef(TableCell))-head-34 MuiTableCell-alignCenter"
              scope="col"
            >
              Remove
            </th>
          </tr>
        </thead>
        <tbody
          class="MuiTableBody-root"
        />
      </table>
    </div>
  </div>
</DocumentFragment>
`;
